templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Are you sure you want to delete this driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'taxi:driver-detail' driver.id %}" class="btn btn-secondary">No</a>
```

Comment:
Ensure that the URL tag uses single quotes consistently for better readability. Change `{% url "taxi:driver-detail" driver.id %}` to `{% url 'taxi:driver-detail' driver.id %}`.